concordium-std/src/impls.rs

```diff
@@ -470,8 +470,13 @@ impl HasActions for Action {
     }
 
     #[inline(always)]
-    fn send(ca: &ContractAddress, receive_name: &str, amount: Amount, parameter: &[u8]) -> Self {
-        let receive_bytes = receive_name.as_bytes();
+    fn send(
+        ca: &ContractAddress,
+        receive_name: &ReceiveName,
```

Comment:
Since `ReceiveName` is already a reference I don't think we should pass a reference to that here.